src/Features/Core/Portable/PdbSourceDocument/PdbSourceDocumentLoaderService.cs

```diff
@@ -26,13 +26,13 @@ internal sealed class PdbSourceDocumentLoaderService : IPdbSourceDocumentLoaderS
         /// Lazy import ISourceLinkService because it can cause debugger 
         /// binaries to be eagerly loaded even if they are never used.
         /// </summary>
-        private readonly Lazy<ISourceLinkService?> _sourceLinkService;
+        private readonly Lazy<ISourceLinkService?>? _sourceLinkService;
         private readonly IPdbSourceDocumentLogger? _logger;
 
         [ImportingConstructor]
         [SuppressMessage("RoslynDiagnosticsReliability", "RS0033:Importing constructor should be [Obsolete]", Justification = "Used in test code")]
         public PdbSourceDocumentLoaderService(
-            [Import(AllowDefault = true)] Lazy<ISourceLinkService?> sourceLinkService,
+            [Import(AllowDefault = true)] Lazy<ISourceLinkService?>? sourceLinkService,
```

Comment:
This was just incorrect. VS Mac doesn't have an implementation of this service, so this would null ref (yet!)

Reply:
@davidwengier Should this be `Lazy<ISourceLinkService>?` instead? Because I don't see how you the lazy could return a null if we're allowing the lazy itself to be null?

Reply:
ahh good point. That is probably how this bug slipped in, the question mark was just put in the wrong place.